### DISCUSSION TITLE
What are the deviation ranges of IHR alarms that classify them as low, mid, or high severity alarms?

### DISCUSSION THREAD

**Discussion by moawnallah** [InternetHealthReport/gsoc#17] [Ideas]
@romain-fontugne For example, in Grip Alarms they have the following business logic for classifying the alarms severity:
<img width="568" alt="Screenshot 2023-08-05 at 12 04 22 AM" src="https://github.com/InternetHealthReport/gsoc/assets/69568555/4607a61d-39ae-4e20-be88-8fd57ba7e951">

**Comment by romain-fontugne** [comment]
For now let's use:
- Low: 20 to 30
- Medium: 31 to 50
- High: over 50

**Reply by moawnallah** [reply]
Resolved in this PR:
https://github.com/InternetHealthReport/ihr-website/pull/554

**Reply by romain-fontugne** [reply]
thanks!
